recipes/newrelic/infrastructure/kubernetes.yml

```diff
@@ -42,7 +42,7 @@ preInstall:
       exit 10
     fi
 
-    K8S_CONFIG_CONTEXTS=$($SUDO $KUBECTL config get-contexts 2>/dev/null --no-headers | wc -l | xargs)
+    K8S_CONFIG_CONTEXTS=$($SUDO $KUBECTL config get-contexts 2>/dev/null | wc -l | xargs)
     if [[ $K8S_CONFIG_CONTEXTS -lt 2 ]]; then
```

Comment:
It might make more sense to leave `--no-headers` in and change the if comparison to `-lt 1`.  We used 2 before to check for a header + at least one row...

Reply:
Thank you!